mtcpclient/calculate_test.go

```diff
@@ -35,23 +36,32 @@ func TestCalculateStdDev(t *testing.T) {
 		},
 	}
 	for _, test := range stdDevScenariosChecks {
-		var gc GroupOfConnections = []tcpclient.Connection{}
+
+		var gc *GroupOfConnections
+		gc = newGroupOfConnections(0)
 
 		var sum int
 		for i, connectionDuration := range test.durationsInSecs {
-			gc = append(gc, tcpclient.NewConnection(i, tcpclient.ConnectionEstablished,
-				time.Duration(connectionDuration)*time.Second))
+			if i%2 == 0 {
+				gc.connections = append(gc.connections, tcpclient.NewConnection(i, tcpclient.ConnectionEstablished,
```

Comment:
the if should just store tcpclient.Connection* in a variable, and then execute a common append. This reduces duplication

Reply:
👍